Description: relax dependency on crate regex
 Needed to match Debian-packaged regex v1.5.5.
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2022-10-04
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/precious-core/Cargo.toml
+++ b/precious-core/Cargo.toml
@@ -21,7 +21,7 @@
 once_cell = "1.6.0"
 precious-exec = { version = "0.3.0", path = "../precious-exec" }
 rayon = "1.5.1"
-regex = "1.6.0"
+regex = "1.5.5"
 serde = { version = "1.0.136", features = ["derive"] }
 tempfile = "3.3.0"
 thiserror = "1.0.30"
--- a/precious-exec/Cargo.toml
+++ b/precious-exec/Cargo.toml
@@ -10,7 +10,7 @@
 [dependencies]
 anyhow = "1.0.53"
 log = "0.4.14"
-regex = "1.6.0"
+regex = "1.5.5"
 thiserror = "1.0.30"
 which = ">= 3.0.0, < 5.0.0"
 
--- a/precious-integration/Cargo.toml
+++ b/precious-integration/Cargo.toml
@@ -9,7 +9,7 @@
 
 [dev-dependencies]
 anyhow = "1.0.53"
-regex = "1.6.0"
+regex = "1.5.5"
 precious-core = { version = "0.3.0", path = "../precious-core" }
 precious-exec = { version = "0.3.0", path = "../precious-exec" }
 precious-testhelper = { version = "0.3.0", path = "../precious-testhelper" }
--- a/precious-testhelper/Cargo.toml
+++ b/precious-testhelper/Cargo.toml
@@ -11,5 +11,5 @@
 anyhow = "1.0.53"
 once_cell = "1.6.0"
 precious-exec = { version = "0.3.0", path = "../precious-exec" }
-regex = "1.6.0"
+regex = "1.5.5"
 tempfile = "3.3.0"
